pyproject.toml

```diff
@@ -20,6 +20,7 @@ django-redis = "^5.0.0"
 wagtail-font-awesome-svg = "0.0.2"
 django-birdbath = "^0.0.5"
 bleach = "^5.0.0"
+django-debug-toolbar = "^3.2.4"
```

Comment:
@TNA-Allan This should be added as a dev dependency only. Try running the following:
```
poetry remove django-debug-toolbar
poetry add django-debug-toolbar --dev
```

Reply:
Thanks very much for this. Done